app/models/metadata.rb

```diff
@@ -65,6 +65,7 @@ def self.metadata_to_json
         meta_attributes[:themes] << attribute.capitalize.gsub('_', ' ') if meta_attributes[attribute]
         meta_attributes.delete(attribute)
       end
+      meta_attributes["metadata"] = meta_attributes["metadata"] ? "http://www.google.com" : nil
```

Comment:
@drewpotter I think you should be able to use `meta.metadata` instead of `meta_attributes["metadata"]` during the assignment. It's shorter :)
Also, you could write the right hand side of the `=` as a function. So we also have the url in there and it is easier to read.

So the URL is something we have not yet given  right? @stacytalbot

Reply:
Perfect, thank you @stacytalbot !